apps/src/applab/DesignModeBox.jsx

```diff
@@ -1,6 +1,5 @@
 /* global $ */
```

Comment:
Ah, I didn't understand that the Gruntfile change made this always allowed as a global. Sounds good. Making `$` always an allowable global sounds good too. 

I know you're not proposing this, but `dashboard` should *not* be an always-allowable global since we want to limit and ideally reduce the number of places where we refer to it.